### DISCUSSION TITLE
postgrest fails when authorization header required by other inline service present

### DISCUSSION THREAD

**Discussion by araji-collibra** [PostgREST/postgrest#2850] [Q&A] [answered]
I am facing an issue where authentication through GCP (to get to postgrest)  requires the same authentication header as the one postgrest uses.

so when I use the correct auth header for my service , postgrest complains:

```
{"code":"PGRST301","details":null,"hint":null,"message":"JWSError JWSInvalidSignature"}
```

are there any workaround ? is there a way to make postgrest ignore the auth token ?

**Comment by laurenceisla** [comment]
There was a similar issue and the OP managed to solve it this way: https://github.com/PostgREST/postgrest/issues/2159#issuecomment-1035404149, although I think using [Nginx](https://postgrest.org/en/v11.1/explanations/nginx.html), as Wolfgang mentioned in the issue, should be a better solution.

**Reply by araji-collibra** [reply, marked as answer]
Thanks, @laurenceisla. Adding nginx in reverse proxy mode did take care of the issue.
